src/database/migrations/V1__addTest.ts

```diff
@@ -0,0 +1,19 @@
+import knex from 'knex';
+
+const TEST_TABLE = 'test';
+
+export async function up(db: knex) {
+  await db.schema
+    .createTable(TEST_TABLE, (table) => {
+      table.uuid('id').primary();
+      table.uuid('user_id').notNullable();
+      table.uuid('test_type_id').notNullable();
+      table.timestamp('creation_time').notNullable().index();
+      table.jsonb('results').nullable();
```

Comment:
Now that the results isn't just the details JSON blob, but also the testerId and creation_tine, I'm wondering if it should be another table.  Not sure...

Reply:
covered in stand up today. Will add testerId and resultsCreationTime.